### PR TITLE
Lfg 187

### PR DESCRIPTION
**Pull request by kusalshrestha**

### CODE REVIEW

File: android/app/keystore.properties [hunk 1]
```diff
@@ -0,0 +1,5 @@
+# temporary data replace for production
+storePassword=leapfroggers
+keyPassword=leapfroggers
+keyAlias=leapfroggers
+storeFile=/Users/Subash/.android/sch.keystore
```
Comment (nishanbajracharya) [inline android/app/keystore.properties]: This is probably not correct.

Comment (surhidamatya) [inline android/app/keystore.properties]: @kusalshrestha please provide a appropriate path to the file with proper naming

File: android/app/src/main/java/com/leapfroggers/MainApplication.java [hunk 2]
```diff
@@ -1,20 +1,16 @@
...
 import com.facebook.react.shell.MainReactPackage;
-import com.facebook.soloader.SoLoader;
 import com.reactnativenavigation.NavigationApplication;
 
 import java.util.Arrays;
 import java.util.List;
 
+import co.apptailor.googlesignin.RNGoogleSigninPackage;
+
+//import com.reactnativenavigation.NavigationReactPackage;
```
Comment (surhidamatya) [inline android/app/src/main/java/com/leapfroggers/MainApplication.java]: @kusalshrestha remove commented part or ad a TODO accordingly

File: app/config/colors.js [hunk 3]
```diff
@@ -2,10 +2,12 @@ const colors = {
   LF_DARK_GRREEN: '#17a555',
   LF_LIGHT_GREEN: '#6cc067',
   GRAY: '#6e6f72',
+  MID_GRAY: '#9E9E9E',
   LIGHT_GRAY: '#c8c8c8',
```
Comment (surhidamatya) [inline app/config/colors.js]: @kusalshrestha it's better to follow single style for naming
`LF_DARK_GRREEN: '#17a555',`
I prefer to follow this
 `MID_GRAY: '#9E9E9E'`

File: app/screens/login/login.js [hunk 4]
```diff
@@ -76,7 +113,7 @@ const GuestUser = {
 
       const user = await GoogleSignin.currentUserAsync();
       if (user) {
-        this.props.validateEmail(user.accessToken);
+        // this.props.validateEmail(user.accessToken);
```
Comment (surhidamatya) [inline app/screens/login/login.js]: @kusalshrestha what's the use of this code?

File: android/app/google-services.json [hunk 5]
```diff
@@ -0,0 +1,48 @@
...
+          "android_client_info": {
+            "package_name": "com.leapfroggers"
+          }
+        },
+        "oauth_client": [
+          {
+            "client_id": "663889381642-1o6p6su9bmrrf45pv96bvtpkdoqguo66.apps.googleusercontent.com",
+            "client_type": 1,
+            "android_info": {
+              "package_name": "com.leapfroggers",
```
Comment (surhidamatya) [inline android/app/google-services.json]: @kusalshrestha we normally don't push this type of file to github.

File: android/app/src/main/java/com/leapfroggers/MainApplication.java (hunk 2 above)
Comment (kusalshrestha) [reply android/app/src/main/java/com/leapfroggers/MainApplication.java]: 🆗 @surhidamatya

File: app/config/colors.js (hunk 3 above)
Comment (kusalshrestha) [reply app/config/colors.js]: 🆗 @surhidamatya

File: android/app/google-services.json (hunk 5 above)
Comment (kusalshrestha) [reply android/app/google-services.json]: 🆗 @surhidamatya

**Comment by kusalshrestha** [comment]
👍

**Pull request merged**